themes/custom/campaign_base/sass/styles.scss

```diff
@@ -7,3 +7,5 @@ $image-path: "/profiles/contrib/cr/themes/custom/campaign_base/images/patternlab
 @import "@comicrelief/pattern-lab/sass/base/_variables";
 @import "@comicrelief/pattern-lab/sass/base/_core";
 @import "@comicrelief/pattern-lab/sass/base/_components";
+
+//test change do not merge me :) 
```

Comment:
@Heleen-cr interesting! let's research.. maybe `develop` wasn't automatically updated after the merge? @Saphyel @adamclark-dev could any of you have a quick look?